docs/getting_started/java/dev_environment/index.md

```diff
@@ -55,7 +55,7 @@ Add the following dependencies to your Maven Project Object Model (POM) configur
     <dependency>
       <groupId>io.temporal</groupId>
       <artifactId>temporal-sdk</artifactId>
-      <version>1.22.4</version>
```

Comment:
These changes need to be done on a different repo. I've gone ahead and done that.

Reply:
And I think it's do-no-harm to merge these into the snipsync as they are now aligned.